tests/drivers/watchdog/wdt_basic_api/testcase.yaml

```diff
@@ -3,7 +3,10 @@ common:
     tags: drivers watchdog
 tests:
   drivers.watchdog:
-    filter: not (CONFIG_WDT_SAM or dt_compat_enabled("st,stm32-window-watchdog") or dt_compat_enabled("st,stm32-watchdog"))
+    filter: >
+      not (CONFIG_WDT_SAM or dt_compat_enabled("st,stm32-window-watchdog")
+       or dt_compat_enabled("st,stm32-watchdog") or CONFIG_SOC_FAMILY_LPC or
+       CONFIG_SOC_SERIES_IMX_RT6XX or CONFIG_SOC_SERIES_IMX_RT5XX)
     platform_exclude: mec15xxevb_assy6853 mimxrt685_evk_cm33 mimxrt595_evk_cm33
```

Comment:
Seems like we don't need the mimxrt685/595 boards in the platform_exclude if we are going to put the family in the not() filter above?

Reply:
yes, Let me remove the platfrom exclude for rt685 and rt595

Reply:
@dleach02 , updated